### PR TITLE
Make installation documentation consistent

### PR DESCRIPTION
**Pull request by 1ec5**
Copyedited installation documentation to make sense to less experienced Xcode users, ensure consistency, remove redundant language, and reflect recent build changes.

/cc @friedbunny @boundsj @tmcw @bleege

### CODE REVIEW

File: platform/ios/INSTALL.md [hunk 1]
```diff
@@ -102,36 +94,42 @@ If your application targets iOS 7.x, you’ll need to install the static framewo
...
+   - SystemConfiguration.framework
+   - libc++.tbd
+   - libsqlite3.tbd
+   - libz.tbd
 
-   - `Settings.bundle` is in your target's *Copy Bundle Resources* build phase. 
+### Configuration
 
-1. Add the following Cocoa Touch frameworks and libraries to the “Linked Frameworks and Libraries” section:
+1. Mapbox vector tiles require a Mapbox account and API access token. In the project editor, select the application target, then go to the Info tab. Under the “Custom iOS Target Properties” section, set `MGLMapboxAccessToken` to your access token. You can obtain an access token from the [Mapbox account page](https://www.mapbox.com/studio/account/tokens/).
```
Comment (bleege) [inline platform/ios/INSTALL.md]: \o/

File: platform/ios/INSTALL.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Integrating the Mapbox iOS SDK into your application
 
-This document explains how to build a development version of Mapbox iOS SDK for the purpose of incorporating it into your own Cocoa Touch application.
+This document explains how to build a development version of Mapbox iOS SDK for the purpose of incorporating it into your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://github.com/mapbox/ios-sdk).
```
Comment (friedbunny) [inline platform/ios/INSTALL.md]: > for the purpose of incorporating

I like this phrase (and realize you did not add it here) but we should take this opportunity to make it simpler.

Perhaps we can change it to:

> development version of Mapbox iOS SDK for **_use in_** your own Cocoa Touch application

Comment (1ec5) [inline platform/ios/INSTALL.md]: Fixed.

**Comment by friedbunny** [comment]
👍

**Pull request merged**